ledger/ledger.go

```diff
@@ -532,6 +532,23 @@ func (l *Ledger) lookupResource(rnd basics.Round, addr basics.Address, aidx basi
 	return res, nil
 }
 
+// LookupKv loads a KV pair from the accounts update
+func (l *Ledger) LookupKv(rnd basics.Round, key string) (*string, error) {
```

Comment:
I have a tough time taking that particular naming convention seriously.  It works fine when the initialism is at the end of the token, but when it's in the middle, it would lead to atrocities like `upsertKVPair` in which P blends in with KV.

I'm not going to die on this hill or anything, but I need at least one more person to say, "Yes, `upsertKVPair` is definitely preferable."

Reply:
I'd probably leave it as is though I'm also not looking to engage in a lengthy discussion.

Rationale:  Since it's neither flagged by a linter nor is it a public API carrying backwards compatibility guarantees, I'd probably follow the path of least resistance.

Personally, I also dislike the suggested convention for a variety of reasons.

Reply:
Resolving - Based on group discussion, we agreed to leave as is until a linter enforces a naming convention.   It's understood that different folks may have applied a different style and/or will change the style.